azkaban-web-server/build.gradle

```diff
@@ -50,27 +71,55 @@ sourceSets {
   }
 }
 
-bower {
-  installBase = 'bower' // <1>
+task movingJsTojsToPackage(dependsOn: ['npm_install']) {
 
-  'moment'('2.14.1'){
-    source 'min/moment.min.js' >> '/'
+  doLast {
+    copy {
+      from "node_modules/later/later.min.js"
+      into 'build/jsToPackage'
```

Comment:
Fixed. I noticed $buidDir is only recognized by double quote. Single quote can not identify $buildDir, do you know why?